### ISSUE TITLE
Unable to load any model.

### ISSUE THREAD

**Issue by nilq** [unitaryai/detoxify#52]
I am unsure whether this is due to being on an M1, but it is my suspicion after having tested with various Python versions satisfying the `>=3.6` requirement on PyPI. It works on my personal laptop running an Arch-based Linux distribution, using the same code and Python 3.9.

---

The following code is being run using `Python 3.9.12`.

```py
__import__('detoxify').Detoxify('original').predict('this does not work')
```

Running this simple test prediction will throw the error below.

```
Traceback (most recent call last):
  File "/opt/homebrew/lib/python3.9/site-packages/torch/serialization.py", line 309, in _check_seekable
    f.seek(f.tell())
AttributeError: 'NoneType' object has no attribute 'seek'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/opt/homebrew/lib/python3.9/site-packages/transformers/modeling_utils.py", line 349, in load_state_dict
    return torch.load(checkpoint_file, map_location="cpu")
  File "/opt/homebrew/lib/python3.9/site-packages/torch/serialization.py", line 699, in load
    with _open_file_like(f, 'rb') as opened_file:
  File "/opt/homebrew/lib/python3.9/site-packages/torch/serialization.py", line 236, in _open_file_like
    return _open_buffer_reader(name_or_buffer)
  File "/opt/homebrew/lib/python3.9/site-packages/torch/serialization.py", line 221, in __init__
    _check_seekable(buffer)
  File "/opt/homebrew/lib/python3.9/site-packages/torch/serialization.py", line 312, in _check_seekable
    raise_err_msg(["seek", "tell"], e)
  File "/opt/homebrew/lib/python3.9/site-packages/torch/serialization.py", line 305, in raise_err_msg
    raise type(e)(msg)
AttributeError: 'NoneType' object has no attribute 'seek'. You can only torch.load from a file that is seekable. Please pre-load the data into a buffer like io.BytesIO and try to load from it instead.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/opt/homebrew/lib/python3.9/site-packages/detoxify/detoxify.py", line 93, in __init__
    self.model, self.tokenizer, self.class_names = load_checkpoint(
  File "/opt/homebrew/lib/python3.9/site-packages/detoxify/detoxify.py", line 49, in load_checkpoint
    model, tokenizer = get_model_and_tokenizer(
  File "/opt/homebrew/lib/python3.9/site-packages/detoxify/detoxify.py", line 19, in get_model_and_tokenizer
    model = model_class.from_pretrained(
  File "/opt/homebrew/lib/python3.9/site-packages/transformers/modeling_utils.py", line 1797, in from_pretrained
    state_dict = load_state_dict(resolved_archive_file)
  File "/opt/homebrew/lib/python3.9/site-packages/transformers/modeling_utils.py", line 352, in load_state_dict
    with open(checkpoint_file) as f:
TypeError: expected str, bytes or os.PathLike object, not NoneType
```

**Comment by laurahanu** [comment]
Hello,

This should have been solved by the latest version of detoxify (v0.5.0) that we released last week. Could you check if you're using it and if not, see if upgrading solves this issue?

**Comment by nilq** [comment]
I have just confirmed that the new version does work. Updated two weeks ago, but just got around to submitting the issue - should have checked for new versions.

Thank you.
